src/Tribe/Views/V2/Assets.php

```diff
@@ -551,21 +551,11 @@ public function should_enqueue_full_styles() {
 	 * @return boolean
 	 */
 	public function should_enqueue_single_event_styles() {
-		// Bail if not V2.
+		// Bail if not Single Event V2.
 		if ( ! tribe_events_single_view_v2_is_enabled() ) {
 			return false;
 		}
 		
-		// Bail if not Single Event.
-		if ( ! tribe( Template_Bootstrap::class )->is_single_event() ) {
-			return false;
-		}
-		
-		// Bail if Block Editor.
-		if ( has_blocks( get_queried_object_id() ) ) {
-			return false;
-		}
```

Comment:
should we not be keeping this? we don't want to enqueue single event styles if it's not a single event page, nor do we want to enqueue if it's a block editor single right?